src/ReactCodeInput.spec.js

```diff
@@ -40,12 +42,22 @@ describe('CodeInputField', () => {
     }
     expect(val.join('')).toEqual(chars);
   });
+  
+  test(`renders component with placeholders: ${placeholders}`, () => {
+    const wrapper = render(<CodeInputField value={numbers} fields={6} type="number" placeholders={placeholders}/>),
```

Comment:
what you think of using singular? `placeholder`

Reply:
great idea